veps/sig-compute/stop-vm-with-pmsuspend-to-disk/stop-vm-with-pmsuspend-to-disk.md

```diff
@@ -0,0 +1,121 @@
+# VEP #NNNN: Your short, descriptive title
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [x] (R) Enhancement issue created, which links to VEP dir in [kubevirt/enhancements] (not the initial VEP PR)
+- [ ] (R) Target version is explicitly mentioned and approved
+- [ ] (R) Graduation criteria filled
+
+## Overview
+
+libvirt have a pmsuspend interface [https://libvirt.org/html/libvirt-libvirt-domain.html#virDomainPMSuspendForDuration], at least for QEMU with flag `target disk` and `duration 0` we can shutdown a domain with  memory to disk. This can release resources. This proposal try to stop vm with `pmsuspend --target disk duration 0`,which can meet specific needs.
+
+## Motivation
+
+Some users wish to shut down running machines to free up resources, but the virtual machine state remains the same when turned on as when turned off.
+
+## Goals
+
+Add a new way to stop vm in kubevirt. which can make virtual machine state remains the same when turned on as when turned off.
+
+## Non Goals
+
+Add pmsuspend to memery to kubevirt（In my opinion, it can implement like pause）
+
+## Definition of Users
+
+<!--
+Who is this feature set intended for
+-->
+
+## User Stories
+
+A user exec `virtctl pmsuspendToDisk vm.name` commend, the vm stopped with memory state saved, exec `virtctl start vm.name` to start the stopped vm.
```

Comment:
Why not `virtctl suspend $VM_NAME` with additional flags to suspend to disk etc?

Avoids leaking the fact that we are using libvirt underneath through our API/commands etc.